packages/edit-site/src/components/block-editor/style.scss

```diff
@@ -106,39 +106,35 @@
 
 	&::after {
 		position: absolute;
-		top: $grid-unit-30;
+		top: $grid-unit-20;
 		left: $grid-unit-05;
 		right: 0;
-		bottom: $grid-unit-30;
+		bottom: $grid-unit-20;
 		content: "";
 		width: $grid-unit-05;
-		background: $gray-600;
+		background-color: rgba($gray-700, 0.4);
 		border-radius: $radius-block-ui;
 	}
 
 	&.is-left {
-		left: -$grid-unit-20;
+		// Subtract half of the handle width to properly center.
+		left: -$grid-unit-20 - ($grid-unit-05 / 2);
 	}
 
 	&.is-right {
-		right: -$grid-unit-20;
+		// Subtract half of the handle width to properly center.
+		right: -$grid-unit-20 - ($grid-unit-05 / 2);
```

Comment:
@juanfra this is causing a deprecation warning:

![Screenshot 2024-05-02 at 12 16 52](https://github.com/WordPress/gutenberg/assets/1204802/0d22a90f-46bc-4389-b2d1-b56d8abe8d32)

You can fix it with right: `-$grid-unit-20 - ($grid-unit-05 * 0.5)`.